tools/docker/Dockerfile.proto-builder

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+FROM golang:1.12-stretch
+
+RUN apt-get update && apt-get install --yes --no-install-recommends \
+	libprotobuf-dev \
+	protobuf-compiler \
+	&& go get -u github.com/containerd/ttrpc/cmd/protoc-gen-gogottrpc \
```

Comment:
Maybe we should remove these binaries from the `deps` target in the repo's top-level Makefile since we now want the proto builds to be done from this container instead.

Reply:
I thought about that but I wasn't sure if we wanted to leave the ability to do builds outside the container? I don't feel strongly about it.

Reply:
+1 for removing them from deps, to make our full build faster.

Reply:
Removed.